LBH.AdultSocialCare.Api/V1/Boundary/DayCarePackageBoundary/Request/DayCarePackageForUpdateRequest.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace LBH.AdultSocialCare.Api.V1.Boundary.DayCarePackageBoundary.Request
+{
+    public class DayCarePackageForUpdateRequest
+    {
+        [Required] public Guid ClientId { get; set; }
+        [Required] public bool IsFixedPeriodOrOngoing { get; set; }
+        [Required] public DateTimeOffset StartDate { get; set; }
+        public DateTimeOffset? EndDate { get; set; }
+        [Required] public bool IsThisAnImmediateService { get; set; }
+        [Required] public bool IsThisUserUnderS117 { get; set; }
+        [Required] public string NeedToAddress { get; set; }
+        [Required] public bool Monday { get; set; }
+        [Required] public bool Tuesday { get; set; }
+        [Required] public bool Wednesday { get; set; }
+        [Required] public bool Thursday { get; set; }
+        [Required] public bool Friday { get; set; }
+        [Required] public bool Saturday { get; set; }
+        [Required] public bool Sunday { get; set; }
+        [Required] public bool TransportNeeded { get; set; }
+        [Required] public bool EscortNeeded { get; set; }
+        [Required] public int TermTimeConsiderationOptionId { get; set; }
+        [Required] public string HowLong { get; set; }
+        [Required] public string HowManyTimesPerMonth { get; set; }
+        [Required] public string OpportunitiesNeedToAddress { get; set; }
+        [Required] public Guid UpdaterId { get; set; }
```

Comment:
To get the full power of [Required] here, you can make the non nullable fields nullable, such as Guid?, otherwise they will default to Guid.Empty for example, and bypass your [Required] attribute

Reply:
> To get the full power of [Required] here, you can make the non nullable fields nullable, such as Guid?, otherwise they will default to Guid.Empty for example, and bypass your [Required] attribute

I need to try this

Reply:
> To get the full power of [Required] here, you can make the non nullable fields nullable, such as Guid?, otherwise they will default to Guid.Empty for example, and bypass your [Required] attribute

Very true. I'll make the update. Thanks